controller/src/beerocks/master/db/db.cpp

```diff
@@ -2494,6 +2494,31 @@ bool db::get_bml_nw_map_update_enable(Socket *sd)
     return false;
 }
 
+bool db::set_bml_topology_update_enable(Socket *sd, bool update_enable)
+{
+    if (sd) {
+        for (auto it = bml_listeners_sockets.begin(); it < bml_listeners_sockets.end(); it++) {
```

Comment:
please use range based loops
```suggestion
        for (const auto &soc : bml_listeners_sockets) {
```